showcase.json

```diff
@@ -2655,6 +2655,20 @@
     "submittableId": "32966927",
     "youtube": "https://www.youtube.com/watch?v=XTDznhHdeOE"
   },
+  {
+    "id": "survey_ease",
+    "name": "SurveyEase",
+    "oneLiner": "Redefining the Survey Landscape",
+    "website": "https://survey-ease.web.app",
```

Comment:
Do you have another link that provides an overview of the product?

Reply:
Unfortunately, this is what we have at the moment. Since the product is not in the state to be live, we havent focused on marketing contents.